jira.yml

```diff
@@ -0,0 +1,56 @@
+- hosts: jira2.liatr.io
+  vars:
+    hostname: 'jira2.liatr.io'
+    jira_data_dir: '/data/jira'
+    postgres_data_dir: '/data/pgsql'
+    postgres_var_lib: 'pgsql96'
+    postgresql_jira_database: 'jira'
+    postgresql_jira_user: 'jira'
+    # please override at command line with --extra-vars
+    postgresql_jira_password: 'CHANGE_ME'
+  roles:
+    - role: liatrio.mount_persist_data
+      become: yes
+    - role: java_internal
+      become: yes
+    - role: postgres_prereqs
+      become: yes
+    - role: geerlingguy.postgresql
+      become: yes
+      ansible_distribution_version: 6.0.0
+      postgresql_version: 9.6
```

Comment:
Saw that Jira supported up to 9.3 officially but that's probably just out of date.  Assuming this works well.

Reply:
Newer versions support the newer version. :)